docs/content/en/functions/path.BaseName.md

```diff
@@ -0,0 +1,24 @@
+---
+title: path.BaseName
+description: BaseName returns the last element of a path, removing the extension if present.
+date: 2022-06-04
+categories: [functions]
+menu:
+  docs:
+    parent: "functions"
+keywords: [path, base]
+signature: ["path.BaseName PATH"]
+relatedfuncs: [path.Base, path.Clean, path.Dir, path.Ext, path.Join, path.Split]
+deprecated: false
+---
+
+If `PATH` is empty, `.` is returned.
+
+**Note:** On Windows, `PATH` is converted to slash (`/`) separators.
+
+```go-html-template
+{{ path.BaseName "a/news.html" }} → "news"
+{{ path.Base "news.html" }} → "news"
```

Comment:
Doesn't Path.Base return news.html instead of news?  That was the purpose of BaseName vs Base.

Reply:
Typo in example. Good catch. Thanks.